examples/verify_addresses_from_csv/verify.py

```diff
@@ -0,0 +1,112 @@
+# Usage
+# python verify.py input.csv
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../../..'))
+
+# TODO: Provide your API key, keep this secure
+lob.api_key = '<insert your api key here>'
+
+# Input check
+if len(sys.argv) < 2:
+    print("Please provide an input CSV file as an argument.")
+    print("usage: python verify.py <CSV_FILE>")
+    sys.exit(1)
+
+input_filename = sys.argv[1]
+
+success_csv_fields = [
+    'address_line1',
+    'address_line2',
+    'address_city',
+    'address_state',
+    'address_zip'
+]
+errors_csv_fields = ['error']
+
+#create the output directory,
+output_dir = os.path.join('.',  'output')
+if not os.path.isdir(output_dir):
+    os.mkdir(output_dir)
+
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+try:
+    output_dir = os.path.join(output_dir, timestamp)
+    os.mkdir(output_dir)
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create output directory. Aborting all sends.')
+    sys.exit(1)
+
+# output csv names
+success_filename = os.path.join(output_dir, 'success.csv')
+errors_filename = os.path.join(output_dir, 'errors.csv')
+
+try:
+    with open(input_filename, 'r') as input, \
+      open(success_filename, 'w') as success, \
+      open(errors_filename, 'w') as errors:
+
+    # print mode to screen
+
+        mode = lob.api_key.split('_')[0]
+        print('Sending letters in ' + mode.upper() + ' mode.')
+
+        input_csv = csv.DictReader(input)
+        errors_csv_fields += input_csv.fieldnames
+
+        success_csv = csv.DictWriter(success, fieldnames=success_csv_fields)
```

Comment:
Did this output style work well for the verified addresses? Do bad ones go to errors?